packages/amplify-e2e-tests/functions/get-ssm-parameter.js

```diff
@@ -0,0 +1,12 @@
+const aws = require('aws-sdk');
```

Comment:
there's already a test function called `retrieve-secret.js` that looks like it does this same thing. Can that be reused instead?

Reply:
We need to use getParameter instead of getParameters, getParameters requires permission to all, but getParameter requires a specific resource.